frontend/src/components/ResultRoutes/ResultRoutes.css

```diff
@@ -41,3 +42,30 @@ IonButton {
 .swiper-div {
     z-index: -10;
 }
+
+.circle-button {
+    --border-radius: 50%;
```

Comment:
For what is --border-radius ? Maybe just border-radius

Reply:
for IonButton